app/assets/javascripts/application.js

```diff
@@ -15,4 +15,22 @@
 //= require turbolinks
 //= require_tree .
 
-$(function(){ $(document).foundation(); });
+window.onload = () => {
+  const listTitle = document.querySelector('.js-list-title')
+  const editListTitle = document.querySelector('.js-edit-list-title')
+  const listTitleForm = document.querySelector('.js-list-title-form')
+  const listFormSubmit = document.querySelector('.js-submit-form')
+
+  editListTitle.addEventListener('click', () => {
+    listTitle.classList.toggle('hidden')
+    listTitleForm.classList.toggle('hidden')
+
+    const input = document.querySelector('#list_name')
```

Comment:
Note: Not that I imagine anyone will be using older browsers such as IE11, but just a note this will break on IE11 and any other browsers that don't support ES6 as const is part of ES6 ( without adding a polyfill to make it compatible with ES5 only browsers ).

Reply:
Good point, it did cross my mind. This app is only for modern browsers 😄